portfolio/src/main/webapp/css/home.css

```diff
@@ -0,0 +1,86 @@
+#my-portrait-img{
```

Comment:
Alpahbetize declarations. This is a great practice for coding and makes it easy to refactor :)